.travis.yml

```diff
@@ -5,6 +5,8 @@ before_install:
 - ./firefox-allow-popups.sh
 - "export DISPLAY=:99.0"
 - "sh -e /etc/init.d/xvfb start"
+- if [ "$TRAVIS_BRANCH" != "master" ] || [ "$TRAVIS_PULL_REQUEST" != false ]; then SAUCE_ACCESS_KEY=$SAUCE_ACCESS_KEY_NOT_MASTER SAUCE_USERNAME=$SAUCE_USERNAME_NOT_MASTER; fi
+- echo "Sauce Labs username is $SAUCE_USERNAME"
```

Comment:
I made the username variables public so I could log them (the access keys are still encrypted).